<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersProjections</class>
 <widget class="QWidget" name="ParametersProjections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="linearTypeCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Linear projection methods:
Naive Bayes: computed separately over the two axes
PCA: Principal Component Analysis
LDA: Linear Discriminant Analysis
Fisher: Fisher Linear Discriminant</string>
   </property>
   <item>
    <property name="text">
     <string>PCA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Means Only (fake LDA)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LDA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fisher LDA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Projection Method</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="projectionButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Projection</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="kernelDegSpin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>40</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Degree of the kernel polynomial
(Polynomial only)</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelDegree">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>45</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Degree</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kernelTypeCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>15</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>kernel function</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polynomial</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RBF</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>45</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelkernel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kernel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kernelWidthSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Width of the kernel (gamma)
RBF and Polynomial only</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
